python/pyproject.toml

```diff
@@ -41,9 +42,7 @@ classifiers = [
 test = [
     "cloudpickle",
     "cudf==24.4.*",
-    "cupy>=12.0.0",
-    "dask",
-    "distributed",
```

Comment:
I think maybe something was missed here? I'd expect to see `dask` and `distributed` drop out and be replaced by `rapids-dask-dependency` in this block.

Reply:
It turns out that the dask bits weren't needed for pure ucxx tests (only distributed-ucxx). The pinning appears in the dependencies of distributed-ucxx. I think we're good here.

Reply:
Or does `ucxx` genuinely not need Dask for its tests?

Sorry not that familiar with this library 😬

Reply:
Hmm...ok. If we do want this here, added a suggestion below. Though no strong feelings either way. Happy to go with whatever was agreed to before

https://github.com/rapidsai/ucxx/pull/174#discussion_r1466735120

Reply:
> I believe that the dask bits aren't needed for pure ucxx tests

Ok got it, thanks.

Reply:
FWIW do see lines like this

So think we may want to include this in the end

https://github.com/rapidsai/ucxx/blob/2249bed6ca09550abb92a6ea9496b0a024697fdd/python/ucxx/utils.py#L29-L30

https://github.com/rapidsai/ucxx/blob/2249bed6ca09550abb92a6ea9496b0a024697fdd/python/ucxx/_lib_async/tests/test_send_recv_two_workers.py#L25

Reply:
Okay, this might have been a misunderstanding on my part. Let's merge the suggestion below and keep it as a test dependency.